mfr/server/static/js/mfr.js

```diff
@@ -68,12 +67,12 @@
      * @param {Object} config Configuration to override the default settings.
      * @param {String} imgName The filename of an image in mfr/server/static/images/ to use as a loading spinner
      */
-    lib.Render = function (id, url, config, imgName) {
+    lib.Render = function (id, url, config, imgName=null) {
```

Comment:
This doesn't look right. Can we just remove this argument `imgName`? But I am not seeing any javascript error from the browser console and rendering works as expected.

Reply:
I wasn't as clear on the jira ticket on this as i should have been. That function is called by the OSF side of things. It asks for a specific spinner ie `cos_logo.png`. The new CSS spinner does not need an `imgName`, but the argument still needs to exist or else things won't render on the OSF (will just throw an error) So i left in `imgName=null` so the OSF can still give it the arg and not break. Also once it is removed it won't break either and then `imgName` can be taken out for good.

Reply:
Using `null` as a default value isn't the way to go though. In javascript, the default value is `undefined` (no need to explicitly set it). And since we no longer use `imgName` in the function, we can set it to `undefined` to overwrite the value that is passed into.

```javascript
lib.Render = function (id, url, config, imgName) {
    // some comment
    imgName = undefined 
}
```

Reply:
I understand now, makes sense to me. Too used to python syntax where 
`lib.Render('12', some_url, some_config)` would throw an error since there is no imgName passed in.

Reply:
Fixed